source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation.py

```diff
@@ -308,10 +308,13 @@ def write_root_pose_to_sim(self, root_pose: torch.Tensor, env_ids: Sequence[int]
         self.root_physx_view.set_root_transforms(root_poses_xyzw, indices=physx_env_ids)
 
     def write_root_velocity_to_sim(self, root_velocity: torch.Tensor, env_ids: Sequence[int] | None = None):
```

Comment:
I suggest we deprecate this function and make it explicit for users to specify the "com" or "link" variants. What do you think?

Reply:
@Mayankm96 do you want to fully deprecate it now or warning like with the root and body_state properties

Reply:
But yes I think the explicit versions are a good idea.